src/operation/list_databases/mod.rs

```diff
@@ -85,5 +85,4 @@ impl Operation for ListDatabases {
 #[derive(Debug, Deserialize)]
 struct ResponseBody {
     databases: Vec<Document>,
-    total_size: Option<i64>,
```

Comment:
This wasn't being used and in fact wasn't even being deserialized, since the server returned this value as a double until 5.0.